### ISSUE TITLE
simplify item/component creation

### ISSUE THREAD

**Issue by es183923** [FelixKratz/SketchyBar#26]
Yabai currently uses the following format for making signals:
```sh
yabai -m signal --add event=<EVENT> action=<ACTION> [label=<LABEL>] [app[!]=<REGEX>] [title[!]=<REGEX>]
```
If SketchyBar could also use a similar way, then the loading of config files might be a bit faster, as well as looking a lot cleaner.
eg:
```sh
sketchybar -m --add example_item \
	label=example \
	type=item \
	location=right \
	associated_space=1,2,3 \
    update_freq=300 \
    script="~/.config/sketchybar/example.sh" \
    click_script="~/.config/sketchybar/example_click.sh"
```
```sh
sketchybar -m --edit example_item \
	label="edited label"
```

**Comment by FelixKratz** [comment]
Yes I want to do this at some point #17.

**Comment by FelixKratz** [comment]
Batch messages are implemented since 05bc579454a6cb0dab09cec70f833fde59bb863f. You can check out the updated README or the updated example sketchybarrc for more info.
